doc/source/performance/jemalloc-profiling.rst

```diff
@@ -0,0 +1,154 @@
+.. _jemalloc-profiling:
+
+=================================================
+Jemalloc Memory Allocation Profiling
+=================================================
+
+Implemented in :rn:`8.0.25-15`, you are able to profile the jemalloc memory allocation.
```

Comment:
I think we need to be clearer and indicate that this is only present and useful if the server is running with the jemalloc library.  @ldonoso ?

Reply:
Fixed. Completely rewritten.

Reply:
Sorry for the delay @georgelorchpercona, yes, I agree, thanks for the change @patrickbirch